src/dates/format-date.js

```diff
@@ -195,4 +195,32 @@ export default function formatDate(date, format, locale = "en") {
 
         return result;
     });
+}
+
+export function formatFormat(date, format, locale = "en") {
```

Comment:
I agree with "dateFormatString".
The "dateFormat" could be misleading, since the result could not be valid date format.